File: robin/python/level1/study5/find_mr_kim.py
def solution(seoul):
    return "".join([("김서방은 " + str(idx) + "에 있다") for idx, val in enumerate(seoul) if val == "Kim"])


def solution1(seoul):
    return "김서방은 {}에 있다".format(seoul.index('Kim'))


if __name__ == '__main__':
    seoul = ["Jane", "Kim"]
    print(solution(seoul))
    print(solution1(seoul))

File: robin/java/src/programmers/level1/study5/FindMrKim.java
package programmers.level1.study5;

public class FindMrKim {
    public static String solution(String[] seoul) {

        int x = 0;
        for (int i = 0; i < seoul.length; i++) {
            if (seoul[i].equals("Kim")) {
                x = i;
                break;
            }
        }
        return "김서방은 " + x + "에 있다";
    }


    public static void main(String[] args) {
        String[] seoul = {"Jane", "Kim"};
        System.out.println(solution(seoul));
    }
}
